src/NewEggAccess/BusinessAccount/Models/Commands/GetFeedStatusCommand.cs

```diff
@@ -0,0 +1,15 @@
+﻿using CuttingEdge.Conditions;
+using NewEggAccess.Configuration;
+
+namespace NewEggAccess.BusinessAccount.Models.Commands
+{
+	public class GetFeedStatusCommand : NewEggCommand
```

Comment:
@maxim-saltanov-skuvault A lot of commands are the same here and in BusinessAccount. Perhaps for those, we can create <command type>CommandBase.cs and then both would inherit from it. This would avoid duplication. We would have flexibility for any future changes to them independently or common changes to the base.

Reply:
@maxim-skuvault When we discussed this task with Jay, it was decided that now there is no need to refactor and I should just repeat the logic for the Business Account, as it is implemented for the Regular Account (so it's mostly duplicated at moment), and do it so as not to affect the current implementation. In the future, we will create a separate ticket and do a general refactoring. I do not see any point in wasting time now on what we in any case then plan to rework for a common implementation for different types of accounts.

Reply:
@maxim-saltanov-skuvault  I was under the impression that we decided not to do a major refactor of the base code, but to use it as the base for Newegg business via inheritance and override values where necessary.  That way we aren't duplicating code and the business classes only need to be changed in the case where they differ.  This way we also keep the benefit of a separate namespace for all business requests.


As an example - this class would simply be:

```suggestion
public class GetFeedStatusCommand : NewEggAccess.Models.Commands.GetFeedStatusCommand
{
	public GetFeedStatusCommand( NewEggConfig config, NewEggCredentials credentials, string payload ) : base( config, credentials, payload )
		{
		}
}
```
I do agree that @maxim-skuvault 's suggestion would be best - as was your initial thought - but as I recall it just sounded like it would take a lot of time to do that now.  I believe the thought was to make a backlog ticket for that refactoring.  If we're to the point where that refactoring would take the same amount of time as simplifying the business account logic via my suggestion - I'm fine with it - but I'll leave that decision up to you.

Reply:
@maxim-skuvault  / @jnewman-sv I thought that the main goal was not to touch the existing code and not to change the existed classes for regular accounts. But I agree that we can simplify BA implementation and to less the duplications of the existing code - so I will revise the code and simplify where possible (as in your examples). But I think we agree that a big changes / refactoring is not required for now?

Reply:
@maxim-saltanov-skuvault Sounds good to tackle refactoring in a separate backlog ticket.